### PR TITLE
Fixes #12681 - Support rubocop 0.35.1 by fixing broken rules

### PR DESCRIPTION
**Pull request by daviddavis**
Build at: http://ci.theforeman.org/job/test_plugin_pull_request/1825/

### CODE REVIEW

File: app/controllers/api/v2/containers_controller.rb [hunk 1]
```diff
@@ -108,7 +108,7 @@ def logs
         render :json => { :logs => Docker::Container.get(@container.uuid)
           .logs(:stdout => (params[:stdout] || true),
                 :stderr => (params[:stderr] || false),
-                :tail   => (params[:tail]   || 100)) }
+                :tail   => (params[:tail] || 100)) }
```
Comment (daviddavis) [inline app/controllers/api/v2/containers_controller.rb]: I think there is a config option to allow the alignment of operators. Interested to hear what other people say here.

File: release/changelog [hunk 2]
```diff
@@ -3,8 +3,7 @@
 # changelog generates two files, Contributors and CHANGELOG with all changes
```
Comment (daviddavis) [inline release/changelog]: We could add this changelog file to rubocop ignore so it stays in sync with the one from foreman core.

**Comment by daviddavis** [comment]
Looks like our tests are hitting this now: https://github.com/defunkt/fakefs/issues/312

**Comment by dLobatog** [comment]
Merged as 7ff105d1beec02c6dfbbd599dd8acf1a73d02f62, thanks @daviddavis . You change doesn't have to do with that and fixes other issues.

**Pull request closed without merging**